Finmer.Game/Gameplay/SceneCombat2.cs

```diff
@@ -172,9 +172,20 @@ private IEnumerable<Participant> GetViableAttackTargets(Participant attacker)
                 .Where(candidate => !candidate.IsSwallowed());
         }
 
+        /// <summary>
+        /// Returns participants that the input participant can attempt to grapple with this turn.
+        /// </summary>
+        /// 
+        private IEnumerable<Participant> GetViableGrappleTargets(Participant opponent) 
+        {
+            return GetViableAttackTargets(opponent)
+                .Where(prey => opponent.Character.CanGrapple(prey.Character));
+        }
+
         /// <summary>
         /// Returns participants that the input participant can attempt to swallow on this turn.
         /// </summary>
+        /// 
```

Comment:
Guessing you didn't intend to add this line. Again, please be sure to go over your changes and view them with a critic's eye, to spot these kinds of accidents / things I'm likely to complain about _before_ you commit them. `git diff` or double-clicking the file in VS' Git integration will show you the changes you've made to a particular file.

It's not that this one comment line is going to break anything, rather I'd like to make the point that not double-checking changes can, in bad cases, result in entirely avoidable breakages.

Reply:
Yep, that was me making space beneath the summary section for the viable prey targets, then realizing I should just create anew and removing most of the newlines, but not all. 

Then I copied the formatting from the nearest available code, which was the portion I altered, so it was a self-perpetuating mistake.

Has been fixed now.